src/views/v2/components/ical-link.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Component: iCal Link
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/components/ical-link.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ * @var obj $ical Object containing iCal data
+ */
+
+if ( empty( $ical->display_link ) ) {
+	return;
+}
+
+?>
+<div class="tribe-events-c-ical tribe-common-b1">
+	<a
+		class="tribe-events-c-ical__link tribe-events-c-small-cta__link tribe-common-cta--thin-alt"
```

Comment:
This shouldn't have the `tribe-events-c-small-cta__link` class right?

Also, if we're using the `--thin-alt` modifier for `tribe-common-cta`, we should be using the `tribe-common-cta` class with it. We also have `tribe-common-anchor`, `tribe-common-anchor-alt`, and `tribe-common-anchor-thin` classes if those are more what you want to match. The `-alt` and `-thin` classes do not need the `tribe-common-anchor` to be present to work, as they are not modifiers.

Reply:
We're using those classes to quickly style up this as a link, but with a font size equal to the previous element (In most views, the navigation). There's no design requirement for this and after testing out different things, this style and size made sense.

Reply:
@paulmskim would you suggest to redefine styles for this new component?

Reply:
The class `tribe-events-c-small-cta__link` only applies `cursor: pointer` to the element. Also by semantics, you should be applying all the `tribe-events-c-small-cta` classes if you're applying the `tribe-events-c-small-cta__link` class to this element. Further, the `<a>` should by default have `cursor: pointer`. So I'm not entirely sure what you're asking here.

Reply:
I understand. So a few things you want to do here:
1. Remove `tribe-events-c-small-cta__link`. As mentioned above, this class doesn't belong here.
2. Remove `tribe-common-cta--thin-alt`. As mentioned above, if you're using this class, you want to use it in conjunction with `tribe-common-cta` as the `--thin-alt` is a modifier. That doesn't belong here either.
3. Add a class `tribe-common-anchor-thin-alt` to common under `base/full/typography/_anchors.pcss`. Use the mixin `anchor-thin-alt`. Add proper overrides. Add `tribe-common-anchor-thin-alt` to this element. As `tribe-common-cta--thin-alt` only applies the `anchor-thin-alt` mixin, rather than using a modifier class meant for a CTA component, using an anchor class meant for applying anchor styles (and not messing with the font size) is what you want to do.

Reply:
Whoops! I just pushed some fixes for this. https://github.com/moderntribe/the-events-calendar/pull/2860/commits/4176b2c7c24d686693f39e88ce869310d80377d3 - I think this way is more clean, I didn't want to re-write a lot of stuff. I think using the mixing for the link can make it directly. Can you check these changes?

Reply:
Just add `tribe-common-anchor-alt` class to this element and nix the styles in https://github.com/moderntribe/the-events-calendar/commit/4176b2c7c24d686693f39e88ce869310d80377d3 then, we're just duplicating CSS by doing this when we already have classes created for this.

Reply:
Right, that's what I originally intended to do, but with the cta.